helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -59,6 +59,12 @@ def _read_group_stage_ids(self, stages, domain, order):
         domain="['|',('team_ids', '=', team_id),('team_ids','=',False)]",
     )
     partner_id = fields.Many2one(comodel_name="res.partner", string="Contact")
+    commercial_partner_id = fields.Many2one(
+        "res.partner",
```

Comment:
Not sure, but for `Many2one` related fields no `comodel_name` needs to be added, could you check?

Reply:
> Not sure, but for `Many2one` related fields no `comodel_name` needs to be added, could you check?

Seems not necessary comodel_name
https://www.odoo.com/documentation/16.0/developer/reference/backend/orm.html#odoo.fields.Many2one

"comodel_name – name of the target model Mandatory except for related or extended fields."

Reply:
Well, then it should be removed IMO

Reply:
done